cedar-policy/src/api.rs

```diff
@@ -2622,6 +2702,51 @@ impl std::fmt::Display for PolicySet {
     }
 }
 
+/// Given a [`PolicyId`] and a [`Policy`], determine if the policy represents a static policy or a
+/// link
+fn is_static_or_link(
+    (id, policy): (PolicyId, Policy),
+) -> Result<Either<est::PolicyEntry, Link>, PolicyToJsonError> {
+    let env = policy
+        .ast
+        .env()
+        .iter()
+        .map(|(id, euid)| (*id, euid.clone()))
+        .collect();
```

Comment:
nit: we only need `env` in the `Some(template_id)` case, so we can move this computation into that branch